api/routes.py

```
@@ -80,7 +93,9 @@ async def analyze_food_by_text(
         return result
     except GeminiServiceException as e:
         logger.error(f"Gemini service error while analyzing food text: {str(e)}")
-        raise HTTPException(status_code=e.status_code, detail=e.message)  # pragma: no cover
+        raise HTTPException(
+            status_code=e.status_code, detail=e.message
+        )  # pragma: no cover
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)